src/contributing.md

```diff
@@ -128,7 +128,7 @@ with one another are marked as "always roll up".
 [rust-highfive]: https://github.com/rust-highfive
 [steveklabnik]: https://github.com/steveklabnik
 [@bors]: https://github.com/bors
-[merge-queue]: https://buildbot2.rust-lang.org/homu/queue/rust
+[merge-queue]: https://bors.rust-lang.org/homu/queue/rust
```

Comment:
This redirects to /queue/rust, so it should be:

```suggestion
[merge-queue]: https://bors.rust-lang.org/queue/rust
```

Reply:
Good point! Fixed them.